src/olympia/promoted/tests/test_utils.py

```diff
@@ -149,3 +150,24 @@ def test_create_stripe_checkout_session_with_custom_rate():
             ],
             customer_email=customer_email,
         )
+
+
+@override_settings(STRIPE_API_WEBHOOK_SECRET="webhook-secret")
+def test_create_stripe_webhook_event():
+    fake_event = "fake-event"
+    payload = "some payload"
+    sig_header = "some sig_header"
+
+    with mock.patch(
+        "olympia.promoted.utils.stripe.Webhook.construct_event"
+    ) as stripe_construct_event:
```

Comment:
Is it possible to have one test that works with some real event data that stripe returns, so it'd be decoded correctly?  i.e. no mocking  (might have to freezegun the time if the signing is time stamped)

Reply:
mm it's tricky because a shared secret is also used to sign the requests.